### PR TITLE
[Docs] some update on major-version-upgrade/upgrade/aws/index.md page

### PR DESCRIPTION
**Pull request by gflorinskaya**
Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4140.s3-website-us-west-2.amazonaws.com/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/

### CODE REVIEW

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md [hunk 1]
```diff
@@ -225,29 +223,29 @@ INFO[2021-11-12T18:22:55-08:00] Created/Updated NVIDIA GPU Feature Discovery Cus
     INFO[2021-11-15T19:59:35-05:00] Run 'export CLUSTER_NAME=konvoy-migration' and follow the rest of the documentation  src="cluster/adopt.go:178"
     ```
 
-1.  Update your environment with the cluster name for use in later steps. Do this by running the shell command from the last line of output in the previous step:
+1.  Update your environment with the cluster name for use in later steps. Do this by running the shell command from the last line of output in the previous step. In this example it is: `export CLUSTER_NAME=konvoy-migration`
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:226]:
```suggestion
1.  Update your environment with the cluster name for use in later steps, by running the shell command from the last line of output in the previous step, `export CLUSTER_NAME=konvoy-migration` in this example:
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md [hunk 2]
```diff
@@ -225,29 +223,29 @@ INFO[2021-11-12T18:22:55-08:00] Created/Updated NVIDIA GPU Feature Discovery Cus
...
 
-    ```sh
-    export CLUSTER_NAME=konvoy-migration
+    ```bash
+    export CLUSTER_NAME=<your cluster name>
     ```
 
-    Then, verify that your environment has the cluster name:
+	Then, verify that your environment has the cluster name:
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:232]:
```suggestion
1.  Verify that your environment has the cluster name:
```

**Comment by Colin1841** [comment]
@gflorinskaya - good idea updating all those SH (haha) - feel free to do that anywhere you feel it'll help the usability of the code examples. I'm not sure why we stopped using that.

**Review by Colin1841 — APPROVED** [review]

**Review by msdolbey — APPROVED** [review]
👍 

BARRRRRY

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md (hunk 2 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md]: This is resolved and changed now.

**Pull request merged**